lib/server.rb

```diff
@@ -21,25 +21,47 @@ def reset_count
     @count = -1
   end
 
-  def format_response(request_lines)
-    reset_count if request_lines[0].include? "/clear_count"
+  def diagnostic_template(normalized_lines)
+    "<pre>" \
+      "Verb: #{normalized_lines[1][0]}\n" \
+      "Path: #{normalized_lines[1][1]}\n" \
+      "Protocol: #{normalized_lines[1][2]}\n" \
+      "Host: #{normalized_lines.last[1].split(":")[0]}\n" \
+      "Port: #{normalized_lines.last[1].split(":")[1]}\n" \
+      "Origin: #{normalized_lines.last[1].split(":")[0]}\n" \
+      "Accept: #{normalized_lines[0][1]}" \
+    "</pre>"
+  end
 
-    response = "<p>Hello, World! (#{count})</p>"
+  def format_response(normalized_lines)
+    response = "#{diagnostic_template(normalized_lines)}"
+    reset_count if normalized_lines[1][1].include? "/clear_count"
     output = "<html><head></head><body>#{response}</body></html>"
     headers = ["http/1.1 200 ok",
-               "date: #{Time.zone.now.strftime('%a, %e %b %Y %H:%M:%S %z')}",
+               "date: #{Time.now.strftime('%a, %e %b %Y %H:%M:%S %z')}",
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.